saleor/channel/models.py

```diff
@@ -32,6 +32,7 @@ class Channel(models.Model):
     )
 
     automatically_confirm_all_new_orders = models.BooleanField(default=True, null=True)
+    allow_to_create_order_without_payment = models.BooleanField(default=False)
```

Comment:
I think, that we could figure out better name here & for Type field. 
maybe something like: 
a) `allow_to_create_unpaid_order_from_checkout`, 
b) `allow_to_create_unpaid_order_from_checkout_complete`
IMO `a` is OK as it explicitly says about creating the order from checkout. but still, it could be misleading, as we also have orderCreateFromCheckout mutation. In that case `b` sounds better.
WDYT? @kadewu  @maarcingebala

Reply:
I guess we could name it `allow_unpaid_orders` or something simple and just mention in the description and the docs in what mutations/cases it applies. What if we changed this behavior in the future? Would we have to change the name of this field then?

Reply:
> What if we changed this behavior in the future? Would we have to change the name of this field then?

`allow_to_create_unpaid_order_from_checkout` is a field dedicated exactly to single action, if we will want to change the behavior or extend it in the future, we could add a new field and/or deprecate the previous one.